docs/conf.py.in

```diff
@@ -26,6 +26,8 @@ THISDIR='@CMAKE_CURRENT_SOURCE_DIR@'
 sys.path.append(os.path.abspath(os.path.join(THISDIR, 'swf-blueprint/docs/python_modules')))
 from substitutions import subst as blueprint
 
+sys.path.append(os.path.abspath(os.path.join(THISDIR, 'swf-blueprint/docs/exts')))
```

Comment:
So why is the python module not in `swf-blueprint/docs/python_modules` like the other one?

Reply:
Because it's not meant to be imported manually. It is a sphinx extension and is only to be used by sphinx itself.

Reply:
Ok, I see, but internally sphinx does the same import I assume because both times we just add that path to the PATH variable, so if it was in python_modules you wouldn't need to add it and we wouldn't need a special directory for it. It is python code and extends the functionality of our swf system, the only difference is how it is imported.

Do you see any downsides to having it in python_modules?

Reply:
Since we already have a special directory for python modules to import it made sense to make a directory for sphinx extensions. We still need to modify the sphinx configuration for every repo in order to add the extension (the extensions variable).

I don't see any direct downsides with moving the extension into python_modules. The only difference is that it improves clarity to have extensions in a separate folder.

Reply:
I'd vote for following best practice here, if there is one. If the Sphinx community proposes and are consistent with putting extensions in a certain place, then I think we should do that too. If not, I also think it makes sense to group it with the rest of the Python code.

Reply:
The only documentation I can find on the topic is to put extensions within the projects directory structure: http://www.sphinx-doc.org/en/stable/ext/thirdparty.html
They then just provide an example where they place extensions in "exts" so I don't see this as a convention. Best practice is simply to keep code within the project directory.

Reply:
Ok, I see. Perhaps move it then, I don't see us having a lot of code in this repo so it might be easier to just gather it all.

Reply:
Ok. I'll create a new PL for the blueprint repo with the changes. After that is merged I'll update this PL with an updated submodule.

Reply:
Ah ok. Cool

Reply:
By PL you mean PR like in Pull Request?

Reply:
Ah yes, my mistake. Fixing that.